src/boss.c

```diff
@@ -474,6 +474,13 @@ void Curse()
   // Every 25th monster becomes a curse. All other monsters are deleted
 
   CurseEnemies();
```

Comment:
I wonder if we shouldn't just skip culling/cursing if we're already in the boss room.

Reply:
That's a good idea! I'll give that a go and see if that breaks anything